src/drivers/boards/av-x-v1/init.c

```diff
@@ -85,6 +85,51 @@
 
 #include "up_internal.h"
 
+/************************************************************************************
```

Comment:
It is probably time to move this into  the common stm32 directory call it board_rc_invert.c and wrap the contents in 
`#if defined(RC_UXART_BASE)`